pkg/driverbuilder/docker.go

```diff
@@ -396,20 +397,43 @@ func forwardLogs(logPipe io.Reader) {
 func multiplexedForwardLogs(logPipe io.Reader) {
 	hdr := make([]byte, 8)
 	for {
+		// Load size of message
 		_, err := logPipe.Read(hdr)
 		if err == io.EOF {
 			break
 		}
 		if err != nil {
 			slog.With("err", err.Error()).Error("log pipe error")
+			return
 		}
 		count := binary.BigEndian.Uint32(hdr[4:])
+
+		// Read message
 		dat := make([]byte, count)
-		_, err = logPipe.Read(dat)
-		if err != nil {
-			slog.With("err", err.Error()).Error("log pipe error")
+		var readCnt int
+		for uint32(readCnt) < count {
+			readBytes, err := logPipe.Read(dat[readCnt:])
+			readCnt += readBytes
+			if err == io.EOF {
+				if uint32(readCnt) == count {
+					break
+				}
+				slog.With("err", err.Error()).Error("log pipe error")
+				return
+			}
+			if err != nil {
+				slog.With("err", err.Error()).Error("log pipe error")
+				return
+			}
+		}
+
+		// Print message line by line
+		lines := strings.Split(string(dat), "\n")
```

Comment:
Instead of printing all the output in a single log, split it line by line and print it.